src/client/mod.rs

```diff
@@ -75,6 +77,11 @@ impl<C: NetworkConnector> Client<C> {
         self.redirect_policy = policy;
     }
 
+    /// Set the UserAgent. Pass `None` to disable.
+    pub fn set_user_agent(&mut self, agent: Option<String>) {
```

Comment:
Hmm, I started looking at other languages, and their built-in Clients don't offer a default user-agent either. The recommended way of setting one is just to set the header. A possible solution to this is `client.get(url).header(UserAgent(someStr)).send()`...